samples/caf_sensor_manager/src/main.c

```diff
@@ -3,20 +3,83 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+
+#include <zephyr.h>
+#include <device.h>
 #include <app_event_manager.h>
+#include <logging/log.h>
+
+#if CONFIG_BOARD_ENABLE_CPUNET
+#include <ipc/ipc_service.h>
+#include <event_manager_proxy.h>
+#endif
+
 
 #define MODULE main
+
 #include <caf/events/module_state_event.h>
+#include <config_event.h>
+#include <caf/events/sensor_data_aggregator_event.h>
 
-#include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE);
 
-
 void main(void)
 {
-	if (app_event_manager_init()) {
-		LOG_ERR("Event manager not initialized");
-	} else {
-		module_set_state(MODULE_STATE_READY);
+	int ret;
+
+	ret = app_event_manager_init();
+	if (ret) {
+		LOG_ERR("Event Manager not initialized, err: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event manager initialized");
+
+#if CONFIG_BOARD_ENABLE_CPUNET
+
+	const struct device *ipc_instance  = DEVICE_DT_GET(DT_NODELABEL(ipc0));
```

Comment:
hide ipc initialization code - I recommend creating a board specific initialization function that will be called here